lib/spack/spack/test/cmd/install.py

```diff
@@ -176,6 +176,19 @@ def test_install_with_source(
                        os.path.join(src, 'configure'))
 
 
+@pytest.mark.parametrize('flag,has_file', [
+    ('--env-variables', True),
+    ('', False)
+])
+def test_install_env_variables(
+    flag, has_file, mock_packages, mock_archive, mock_fetch, config, install_mockery
+):
+    spec = Spec('libdwarf')
```

Comment:
Can we have a test that if the flag isn't there the file does not exist?

Reply:
It's parametrized and testing that too :)

Reply:
Totally missed that, thanks!